pangeo-notebook/environment.yml

```diff
@@ -43,6 +43,7 @@ dependencies:
  - gsw
  - h5netcdf
  - h5py
+ - hdf5
```

Comment:
This line can be removed, because we already depend on `netcdf4` -> `libnetcdf` -> `hdf5`.

```suggestion
```